packages/api/src/utils/env.js

```diff
@@ -0,0 +1,3 @@
+export function magicTestModeFromEnv (env) {
+  return Boolean(env?.NEXT_PUBLIC_MAGIC_TESTMODE_ENABLED)
```

Comment:
The only time test mode should be set is if the user provides the string value `"true"` so let's say it loud and clear here:
```suggestion
  return env.NEXT_PUBLIC_MAGIC_TESTMODE_ENABLED === "true"
```
the optional chaining would guard against folks using this function but passing no arguments, which feels like it would be a programming mistake that should throw rather than be dodged.

Reply:
obvs "string value true" is gross, but these are env vars. we are deep in the string mines.

Reply:
I think you commented on out outdated part. I updated `magicTestModeFromEnv` to handle values like `"false"` as falsy (and values like `"1"` as truthy) via https://github.com/web3-storage/web3.storage/commit/754e9eb0b60aad56279e9d628267e823cf28bbef